ldclient/testing/impl/datasystem/test_fdv2_datasystem.py

```diff
@@ -432,3 +435,112 @@ def test_fdv2_stays_on_fdv1_after_fallback():
     store = fdv2.store
     flag = store.get(FEATURES, "fdv1-flag", lambda x: x)
     assert flag is not None
+
+
+def test_fdv2_with_file_to_polling_initializers():
```

Comment:
These tests reference polling, but they are using the test data sources instead. So we should probably update the language.